tests/Data/ProgrammesDb/EntityRepository/PromotionRepository/FindPromotionsByPidTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+
+use BBC\ProgrammesPagesService\Domain\ValueObject\Pid;
+use Tests\BBC\ProgrammesPagesService\AbstractDatabaseTest;
+
+/**
+ * @coversDefaultClass BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\PromotionRepository
```

Comment:
if I use later @covers notation to indicate the function that you are covering, then yes. Otherwise phpunit fires warnings (complaining about notation, no about tests)